pom.xml

```
@@ -97,7 +97,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>script-security</artifactId>
-            <version>1.58</version>
+            <version>1.60-rc860.42e743108fee</version>
```

1.61 was just released.
```suggestion
            <version>1.61</version>
```